hathor/indexes/rocksdb_timestamp_index.py

```diff
@@ -38,13 +38,16 @@ class RocksDBTimestampIndex(TimestampIndex, RocksDBIndexUtils):
     It works nicely because rocksdb uses a tree sorted by key under the hood.
     """
 
-    def __init__(self, db: 'rocksdb.DB', *, cf_name: bytes) -> None:
-        RocksDBIndexUtils.__init__(self, db)
+    def __init__(self, db: 'rocksdb.DB', name: str) -> None:
         self.log = logger.new()
+        RocksDBIndexUtils.__init__(self, db, f'timestamp-sorted-{name}'.encode())
```

Comment:
Why is it different from `get_db_name()`?

Reply:
This name pattern already exist, it doesn't need to be the same. We only use `get_db_name` for generating the name of the attribute used to track an index state, and this pattern is used to name the column-family that the implementation of the index uses. I don't think the two should be tied, an index could in theory use multiple column-families (which is already a refactor that we discussed that could improve the rocksdb-deps-index implementation).

Maybe `get_db_name` could be called `get_attr_name` or `get_index_state_name` or something long those lines.

Reply:
I guess it's ok. We can refactor it later if needed. I was just wondering why they were different.